Sources/StreamChatUI/Utils/ComponentsProvider_Tests.swift

```diff
@@ -101,7 +101,7 @@ private class TestViewWithExtraData<ExtraData: ExtraDataTypes>: UIView, Componen
 
 private class TestViewWithExtraDataController<ExtraData: ExtraDataTypes>: UIViewController, ComponentsProvider {
     let subView = TestViewWithExtraData<ExtraData>()
-    lazy var TestCreateChannelButton = components.createChannelButton.init()
+    lazy var TestCreateChannelButton = components.galleryView.init()
```

Comment:
let's make it `TestGalleryView` so the naming reflect the component type

Reply:
Missed this one, I'll fix this in the next one. Thanks!